### DISCUSSION TITLE
Improved keyword obfuscation

### DISCUSSION THREAD

**Discussion by AdalynBlack** [ThioJoe/YT-Spammer-Purge#577] [Ideas]
The keywords being obfuscated how they are is great for preventing low level scammers from being able to figure out the filter, but more advanced scammers might get through still. To prevent this from becoming a problem, it might be worth implementing a better system for obfuscating the keywords. The first thought that I had is, rather than decoding the keywords and comparing it to the plain text, instead encrypt the plaintext with something like a sha sum, then compare it to the sums of the keywords. Doing that would mean that, in order to get the keywords, you would need to brute force every single sum, which is completely unreasonable, and definitely would not be cost efficient for scammers.

**Comment by ThioJoe** [comment]
Hm yea maybe something the other way round is possible. But really the intention _is_ to only prevent the low level scammers anyway
